### PR TITLE
payment method enum members

### PR DESCRIPTION
**Pull request by D-Workz**
https://schema.org/PaymentMethod

Enumeration members of goodrelations.

### CODE REVIEW

File: schema/paymentMethods/paymentMethods.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
+{
+  "@context": {
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "schema": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@graph": [
+    {
+      "@id": "goodrelations:ByBankTransferInAdvance",
```
Comment (eliaskaerle) [inline schema/paymentMethods/paymentMethods.json]: @D-Workz , where do you define the goordrelations URI? should be in the context I suppose?

**Comment by eliaskaerle** [comment]
please add the namespace URI for goodrelations to the context. rest looks good.

File: schema/paymentMethods/paymentMethods.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
+{
+  "@context": {
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "schema": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "goodrelations":"http://purl.org/goodrelations/v1#"
```
Comment (D-Workz) [inline schema/paymentMethods/paymentMethods.json]: Done ;)

Comment (eliaskaerle) [reply schema/paymentMethods/paymentMethods.json]: Great!

File: schema/paymentMethods/paymentMethods.json (hunk 1 above)
Comment (eliaskaerle) [reply schema/paymentMethods/paymentMethods.json]: solved

**Review by eliaskaerle — APPROVED** [review]
looks good!

**Pull request merged**